source/assets/scripts/main.js

```diff
@@ -131,6 +140,20 @@ function bindRecipeCards(query) {
     );
     recipeViewersWrapper.classList.remove("shown");
 
+    // Display the search filter
+    searchFilter.classList.add("shown");
+
+    // Hide the homepage-section
+    const homepage = document.querySelector(
+      ".section-home-page"
+    )
+    homepage.classList.remove("shown");
+
+    // Display navbar search bar
+    const navSearchDisplay = document.querySelector(
+      ".nav-search-bar"
+    )
+    navSearchDisplay.classList.remove("hidden");
```

Comment:
With the other comment, here you would add `navSearchDisplay.classList.add('shown')`.